website/package.json

```diff
@@ -22,18 +22,18 @@
     "react-feather": "2.0.9",
     "react-frame-component": "4.1.3",
     "simpl-schema": "^1.0.0 || ^0.0.4",
-    "uniforms": "^3.0.0",
-    "uniforms-antd": "^3.0.0",
-    "uniforms-bootstrap3": "^3.0.0",
-    "uniforms-bootstrap4": "^3.0.0",
-    "uniforms-bootstrap5": "^3.0.0",
-    "uniforms-bridge-graphql": "^3.0.0",
-    "uniforms-bridge-json-schema": "^3.0.0",
-    "uniforms-bridge-simple-schema": "^3.0.0",
-    "uniforms-bridge-simple-schema-2": "^3.0.0",
-    "uniforms-material": "^3.0.0",
-    "uniforms-semantic": "^3.0.0",
-    "uniforms-unstyled": "^3.0.0",
+    "uniforms": "3.7.0",
+    "uniforms-antd": "3.7.0",
+    "uniforms-bootstrap3": "3.7.0",
+    "uniforms-bootstrap4": "3.7.0",
+    "uniforms-bootstrap5": "3.7.0",
+    "uniforms-bridge-graphql": "3.7.0",
+    "uniforms-bridge-json-schema": "3.7.0",
+    "uniforms-bridge-simple-schema": "3.7.0",
+    "uniforms-bridge-simple-schema-2": "3.7.0",
+    "uniforms-material": "3.7.0",
+    "uniforms-semantic": "3.7.0",
+    "uniforms-unstyled": "3.7.0",
```

Comment:
So I guess since now we have to update it manually on every publication of uniforms. What is the cause of this change?

Reply:
I'm convinced that Lerna will actually update them on release.